src/partials/algolia-script.hbs

```diff
@@ -72,7 +72,7 @@ window.addEventListener('DOMContentLoaded', function() {
 
   autocomplete({
     container: '#autocomplete',
-    placeholder: 'Search for docs',
+    placeholder: 'Search',
```

Comment:
```suggestion
    placeholder: 'AI Search',
```

Reply:
This is the placeholder for Algolia search 😄

Reply:
It doesn't just search docs anymore.

Reply:
Ahhh got it!